src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -93,7 +93,17 @@ export interface ModelInformationDNE {
 export interface ModelInformationER {
     indicatorsModifier: Record<string, number>;
     deprecatedUrlToFieldValueSize: number;
+    /**
+     * The sample of the top queries for which the model could recommend items for each listed context key.
+     *
+     * @deprecated consider using `contextCandidateExamples` which provides the same data but not prefixed with
+     * `c_context` and no `visit` suffix
+     */
     candidateExamples: Record<string, string[]>;

```

Comment:
The interface `ModelInformationER` seems to be only used by a get call in the `platform-client`.

So, the good thing is that we don't do a breaking change here yay. However then, the questions:
 - is `candidateExamples` still returned by the Rest API?
 - is the arg on death-row (i.e. slated for deprecation)?
If so, I would maybe consider ripping the band-aid straight off and remove it.

But they are both very valid options, so I let you judge, you have surely more context (but from the CDX, deprecate or breaking change we're fine with both in the present case)

Reply:
That seems fine to me 👍